src/osbuild-manifests/platform.kubevirt.ipp.yaml

```diff
@@ -98,7 +98,10 @@ pipelines:
               mpp-if: arch == 'aarch64'
               then: arm64
               else: 
-                mpp-format-string: '{arch}'
+                mpp-if: arch == 's390x'
+                then: s390x
```

Comment:
Is this needed? I mean the default would return `s390x anyway right?`

Reply:
we probably need to pass some value here, so I would say yes?

I know for this particular one there's no "mapping" that's happening, but I don't think it hurts.

Reply:
oh I see now the else below..

yeah, the `else: mpp-format-string: '{arch}'` should already cover this.